client/src/components/Feed/Post.js

```diff
@@ -339,21 +339,27 @@ const Post = ({
 
   const renderHeader = (
     <Card.Header
-      title={post?.author?.name}
+      title={
+        <div className="title-Wrapper">
```

Comment:
need to update to Kebab case here on line 343 too!

Reply:
Fixed!